src/main/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -136,68 +129,241 @@
+         @NoMergingIsDerived
+         public double PCT_40X;
         /** The fraction of bases that attained at least 50X sequence coverage in post-filtering bases. */
-        public double PCT_50X;
+         @NoMergingIsDerived
+         public double PCT_50X;
         /** The fraction of bases that attained at least 60X sequence coverage in post-filtering bases. */
-        public double PCT_60X;
+         @NoMergingIsDerived
+         public double PCT_60X;
         /** The fraction of bases that attained at least 70X sequence coverage in post-filtering bases. */
-        public double PCT_70X;
+         @NoMergingIsDerived
+         public double PCT_70X;
         /** The fraction of bases that attained at least 80X sequence coverage in post-filtering bases. */
-        public double PCT_80X;
+         @NoMergingIsDerived
+         public double PCT_80X;
         /** The fraction of bases that attained at least 90X sequence coverage in post-filtering bases. */
-        public double PCT_90X;
+         @NoMergingIsDerived
+         public double PCT_90X;
         /** The fraction of bases that attained at least 100X sequence coverage in post-filtering bases. */
-        public double PCT_100X;
+         @NoMergingIsDerived
+         public double PCT_100X;
 
         /** The theoretical HET SNP sensitivity. */
+        @NoMergingIsDerived
         public double HET_SNP_SENSITIVITY;
 
         /** The Phred Scaled Q Score of the theoretical HET SNP sensitivity. */
+        @NoMergingIsDerived
         public double HET_SNP_Q;
+
+        /**
+         * Merges the various PCT_EXC_* metrics.
+         * @param other metric to merge into this one.
+         */
+        @Override
+        public void merge(final MergeableMetricBase other) {
+            final WgsMetrics otherMetric = (WgsMetrics) other;
+
+            if (depthHistogram == null || otherMetric.depthHistogram == null) throw new PicardException("Depth histogram is required when deriving metrics.");
+
+            // NB:
+            // Since: PCT_EXC_TOTAL     = (totalWithExcludes - thisMetricTotal) / totalWithExcludes;
+            // Thus:  totalWithExcludes = total / (1 - PCT_EXC_TOTAL)
+            // Proof: Exercise is left to the reader.
+            final long thisMetricTotal        = (long) depthHistogram.getSum();
+            final long otherMetricTotal       = (long) otherMetric.depthHistogram.getSum();
+            final long total                  = thisMetricTotal + otherMetricTotal;
+            final long thisTotalWithExcludes  = (long) (thisMetricTotal / (1.0 - PCT_EXC_TOTAL));
+            final long otherTotalWithExcludes = (long) (otherMetricTotal / (1.0 - otherMetric.PCT_EXC_TOTAL));
+            final double totalWithExcludes    = thisTotalWithExcludes + otherTotalWithExcludes;
+
+            if (0 < totalWithExcludes) PCT_EXC_DUPE     = (PCT_EXC_DUPE * thisTotalWithExcludes + otherMetric.PCT_EXC_DUPE * otherTotalWithExcludes) / totalWithExcludes;
```

Comment:
I guess it can't be done currently. 

what I had envisioned for cases like this is that we would have private or protected members (so they do not get output in the metrics) that are `@MergedByAdding`. I think I have an implementation of that somewhere, but it's dependent on some changes that I didn't get around to doing (yet!!)...